src/components/EnterpriseApp/index.jsx

```diff
@@ -190,6 +191,7 @@ EnterpriseApp.propTypes = {
       enterpriseSlug: PropTypes.string.isRequired,
     }).isRequired,
   }).isRequired,
+  enterpriseId: PropTypes.string,
```

Comment:
Should the enable code management & subscription management screens also have default props of false (to match the sidebar)?